### PR TITLE
Cossack Coats and Schnapps

### PR DESCRIPTION
**Pull request by Wildwestmars**
## About The Pull Request
<details>
<summary>
	Three new coats for Jaegers to flaunt how connected they are to being trench-coat wearing weirdies in the woods. Also Booze!
</summary>
<hr>

The first two coats are for everyone, doubtless that the jaeger riding coat will find a lot of use for hunters due to it's appearance.
The most complex coat, the hussar coat, is locked off to security due to it's very unique color scheme that make's it look out of place on a non-militia or marshal, and it's armor value, which is the same as the default armor vest.

Sneaking it's way in is a new alcohol bottle and recipe, the bottle can only be obtained through hacking the Booze-O-Mat. The vintage is sold by Lonestar Shipping and meant exclusively for export to Krios.
	
<hr>
</details>

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -20,3 +20,4 @@ __pycache__
 .vscode
 .history
 maps/*/backup/
+icons/inventory/New Text Document.txt
```
Comment (Trilbyspaceclone) [inline .gitignore:23]: ?

File: code/modules/clothing/suits/greatcoat.dm [hunk 2]
```diff
@@ -163,4 +163,38 @@
...
+	siemens_coefficient = 0.7
+
+/obj/item/clothing/suit/greatcoat/cossackcoat/edgelord
+	name = "jaeger service coat"
+	desc = " A tailored Jaeger dress-coat, kept to collect dust in a wardrobe all-year until the time of a village's ceremony or celebration. It is rigidly stitched and chokes the torso and arms to stiffen the chest and announce one's posture."
+	icon_state = "cossackcoat_edgelord"
+	item_state = "cossackcoat_edgelord"
+
+/obj/item/clothing/suit/greatcoat/cossackarmoredcoat
+    name = "jaeger hussar coat"
```
Comment (Trilbyspaceclone) [inline code/modules/clothing/suits/greatcoat.dm:187]: All of these are spaces rather then tabs

File: .gitignore (hunk 1 above)
Comment (Wildwestmars) [reply .gitignore:23]: I- suck at using github

File: code/modules/clothing/suits/greatcoat.dm (hunk 2 above)
Comment (Wildwestmars) [reply code/modules/clothing/suits/greatcoat.dm:187]: Understood, I'll try to stop doing that

Comment (Wildwestmars) [reply code/modules/clothing/suits/greatcoat.dm]: fixed

**Pull request merged**